docs/docs/30-how-to-guides/20-managing-credentials.md

```diff
@@ -399,8 +398,32 @@ to set up EKS Pod Identity in your EKS cluster and assign an IAM role to the
 `kargo-controller` `ServiceAccount` within the `Namespace` to which Kargo is (or
 will be) installed.
 
-At this point, an IAM role will be associated with the Kargo _controller_,
-however, that controller acts on behalf of multiple Kargo projects, each of
+##### IRSA
+
+Using IRSA is also possible as a fallback to the default of EKS Pod Identity,  
+
+First, follow
+[this overview](https://docs.aws.amazon.com/eks/latest/userguide/associate-service-account-role.html)
+to set up IRSA in your EKS cluster and assign an IAM role to the
+`kargo-controller` `ServiceAccount` within the `Namespace` to which Kargo is (or
+will be) installed. Ensure that the IAM role is allowed to assume other roles.
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "sts:AssumeRole",
+            "Resource": "arn:aws:iam::ACCOUNT_ID:role//kargo-project-*"
+        }
+    ]
+}
+```
```

Comment:
I'm not positive that this is the best way to do this. (Trust policies may be better ways to say what Roles can be assumed by other Roles). But rather than get caught up in that...

imho, we don't want our docs to get overly mired in AWS details. Their docs give better guidance on topics such as these than we are able to anyway. This is why, in the Pod Identity section, I deliberately gave minimal, but sufficient info about the Kargo end of things and then referred readers to AWS docs for further details:

> At this point, an IAM role will be associated with the Kargo controller, however, that controller acts on behalf of multiple Kargo projects, each of which may require access to different ECR repositories. To account for this, when Kargo attempts to access an ECR repository on behalf of a specific project, it will first attempt to [assume an IAM role](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html) specific to that project. The name of the role it attempts to assume will always be of the form kargo-project-<project name>. It is this role that should be granted read-only access to applicable ECR repositories.

Reply:
I modified the docs to be a bit more compact, but that the same time tried to also give a few minor pointers in regards to role assumption. What do you think?